.github/workflows/tests.yml

```diff
@@ -15,6 +15,11 @@ on:
        - '.github/workflows/*.yml'
   workflow_dispatch:
 
+# Cancel jobs on the same ref if a new one is triggered
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
`github.ref` is the branch of this PR, right?

Reply:
yes. any branch (incl main)